bin/link-all.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+"use strict";
+
+const fs = require('fs');
+const globSync = require('glob').sync;
+const execSync = require('child_process').execSync;
+const cwd = process.cwd();
+
+const LINK_COMMAND = 'yarn link';
+
+globSync('dist/**/@glimmer/*/', { cwd })
```

Comment:
small nit-pick, and should not be considered a blocker.

I would recommend using [walkSync](https://www.npmjs.com/package/walk-sync) here, as it prevents duplicating stating of `dir` (as during traversal we must already stat)

```js
const walkSync = require('walk-sync');
walkSync.entries('dist/**/@glimmer/*').forEach( entry => {
  if (entry.isDirectory()) {
    // ....
  }
});
``